vuepress/docs/next/docs/curate/bundle-details.md

```diff
@@ -184,6 +185,16 @@ See the [Plugin Environment Variables](../../tutorials/devops/plugin-environment
     }
    ]
 ```
+#### Configure a Path for Static Assets
+To configure your micro frontend with access to static assets, Entando provides two paths, one for widgets and another for EPCs, defined in a custom element.  
+
+* For widgets: `window.entando?.widgets['YOUR-MFE']?.basePath;`
+
+* For EPCs: `window.entando?.epc['simple-mfe']?.basePath;`
+
+[See instructions on configuring a custom element](../../tutorials/create/mfe/react.md#configure-the-custom-element). 
```

Comment:
See instructions on configuring a custom element -> See instructions on setting the path in a React MFE.